src/test/java/net/openhft/chronicle/network/NetworkTestCommon.java

```diff
@@ -78,14 +78,14 @@ public void resetClock() {
     public void afterChecks() {
         CleaningThread.performCleanup(Thread.currentThread());
 
+        TCPRegistry.reset();
```

Comment:
why move up here? Should we call this before AbstractCloseable.waitForCloseablesToClose etc.?

Reply:
Hmm I thought that was what complained when the Closeables weren't closed but it looks like I might be able to leave it where it was. I'll revert that.